packages/frontend/src/Routes.tsx

```diff
@@ -46,6 +47,18 @@ import UserActivity from './pages/UserActivity';
 import VerifyEmailPage from './pages/VerifyEmail';
 import ViewSqlChart from './pages/ViewSqlChart';
 
+const DashboardPageWrapper: FC<{ keyParam: string }> = ({ keyParam }) => {
+    const params = useParams();
```

Comment:
Extract page component so we can use params hooks rather than `props.match.params`